crit/cmd/cli.go

```diff
@@ -33,7 +33,8 @@ var decodeCmd = &cobra.Command{
 	Long: `Convert the input binary image to JSON and writes it to a file.
 If no output file is provided, the JSON is printed to stdout.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		c = crit.New(inputFilePath, outputFilePath, inputDirPath, exploreType, pretty, noPayload)
+		c = crit.New(inputFilePath, outputFilePath,
+			inputDirPath, exploreType, pretty, noPayload)
```

Comment:
Would it be possible to move the changes that split long lines in a separate commit?
It would make it much easier to review.